### ISSUE TITLE
Cannot read properties of undefined (reading 'replace')

### ISSUE THREAD

**Issue by evanstern** [junedomingo/react-native-rename#185]
**Describe the bug**
Running `npx react-native-rename "New Name"` results in an error after the `ios/XXX/Info.plist` reports itself as UPDATED.

The error is 

```
TypeError: Cannot read properties of undefined (reading 'replace')
   at Cn (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/react-native-rename/lib/index.js:50:3569)
    at vn.<anonymous> (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/react-native-rename/lib/index.js:61:1031)
    at async vn.parseAsync (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/react-native-rename/lib/index.js:10:4063)
```

**To Reproduce**
1. Create a new project with `npx react-native init`
2. Enter the new project directory
3. Attempt to rename the project using `npx react-native-rename "New Name"`
4. Updates occur until after Info.plist and then an error is displayed

**Expected behavior**
It should not throw an error

**Environment (please complete the following information):**
 - operating system: MacOS Monterey
 - npm version: 8.11.0
 - node version: 16.15.1
 - react-native-rename version: 3.2.7
 - react-native version: 0.71.0

**Comment by shaquibimdad** [comment]
I am facing the same error

**Comment by junedomingo** [comment]
Fixed in v3.2.8, cheers!
